### ISSUE TITLE
单层全连接snn相关

### ISSUE THREAD

**Issue by Linxiang-Wang** [fangwei123456/spikingjelly#499]
Hello， I noticed that in the tutorial, a single-layer fully connected SNN is used for recognizing MNIST instead of N-MNIST. Why is that?At the same time, I also tried using a single-layer fully connected SNN for DVS Gesture recognition, but the accuracy only reached 72%. Is this a normal phenomenon?

**Comment by Linxiang-Wang** [comment]
I changed the input dataset of the single-layer fully connected SNN to N-MNIST, and it can still undergo training. However, is the Poisson encoding redundant in this case?

**Comment by fangwei123456** [comment]
>  for recognizing MNIST instead of N-MNIST. Why is that?

Because users are familiar with MNSIT. To use N-MNIST, users have to learn how to process neuromorphic datasets first.

> but the accuracy only reached 72%. Is this a normal phenomenon?

It is as expected.

> is the Poisson encoding redundant in this case?

No, because some new users may not agree to input data to SNNs directly.

**Comment by Linxiang-Wang** [comment]
I see. So, what you mean is that the example of the single-layer fully connected SNN recognizing MNIST with MNIST + Poisson encoding actually demonstrates a process for handling static datasets?

**Comment by fangwei123456** [comment]
Yes. In the next tutorial, the encoder is removed and the image is sent to the SNN directly.
